### PR TITLE
[KYUUBI #1769][FOLLOWUP] Some cleanups for log4j properties

### PR DESCRIPTION
**Pull request by turboFei**
### _Why are the changes needed?_

This is a followup of #1769. Cleanup the log4j properties files and using log4j2.properties for UT.

### _How was this patch tested?_

Passed UT.

### CODE REVIEW

**Comment by yaooqinn** [comment]
and the licenses?

**Comment by turboFei** [comment]
> and the licenses?

Sorry, which file should I modify?

**Comment by yaooqinn** [comment]
LICENSE-binary

**Review by yaooqinn — APPROVED** [review]

**Review by pan3793 — APPROVED** [review]

File: LICENSE-binary [hunk 1]
```diff
@@ -247,7 +247,6 @@ org.apache.thrift:libfb303
 org.apache.thrift:libthrift
 org.webjars:swagger-ui
 org.yaml:snakeyaml
-log4j:log4j
```
Comment (yaooqinn) [inline LICENSE-binary:250]: it still on our `depenencyList` can we also remove it there?

Comment (turboFei) [reply LICENSE-binary:250]:
```
    <dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>org.apache.kyuubi</groupId>
                <artifactId>kyuubi-common_${scala.binary.version}</artifactId>
                <version>${project.version}</version>
                <exclusions>
                    <exclusion>
                        <groupId>log4j</groupId>
                        <artifactId>log4j</artifactId>
                    </exclusion>
                </exclusions>
            </dependency>

...
    </dependencyManagement>
```

File: pom.xml [hunk 2]
```diff
@@ -240,6 +240,18 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>org.apache.kyuubi</groupId>
```
Comment (yaooqinn) [inline pom.xml:244]: emm?

Comment (turboFei) [reply pom.xml:244]: Without this, the log4j-1.2.17 dependency will be transferred.

Comment (yaooqinn) [reply pom.xml:244]: to be clear, Do you mean that the common module needs log4j-1.2.17 but the server module not? so you need to exclustion?

Comment (turboFei) [reply pom.xml:244]: The kyuubi-common module is depended by other modules, such as kyuubi_assembly.

For command, `mvn dependency:tree -pl :kyuubi-common_2.12|grep 1.2.17`, the result is empty.


But if you execute below command:
```
mvn dependency:tree |grep  -C 20 1.2.17
```

You can find that the log4j-1.2.17 is from kyuubi-common module.

Comment (turboFei) [reply pom.xml:244]: So, I think there is some dependency transfer issue.


And declare the kyuubi-common in dependencyManagement can help fix this issue.

Comment (yaooqinn) [reply pom.xml:244]: why not exclude it in common？

Comment (turboFei) [reply pom.xml:244]: let me have a try.

**Comment by turboFei** [comment]
thanks, merged to master

**Pull request closed without merging**